add-on/chrome/bootstrap.js

```diff
@@ -187,15 +187,8 @@ var WindowListener = {
             this.LoopAPI.initialize();
 
             let anchor = event ? event.target : this.toolbarButton.anchor;
-            let setHeight = 410;
-            if (gBrowser.selectedBrowser.getAttribute("remote") === "true") {
-              setHeight = 262;
-            }
             this.PanelFrame.showPopup(window, anchor,
-              "loop", null, "about:looppanel",
-              // Loop wants a fixed size for the panel. This also stops it dynamically resizing.
-              { width: 330, height: setHeight },
-              callback);
+              "loop", null, "about:looppanel", null, callback);
```

Comment:
Please run the mochitests to make sure this doesn't break anything, and adjust if necessary.

Note to self: need to review param usage.